File: lang/tks-amiga2/tks-amiga/gg/hello.pl
$i=int(0);
$x=0,$y=0.1,$z=0.11,$w=0.0;
$s="";
while($i++<100000)
{
    $w=$w+$x+$y+$z;
    $s.="ab";
}
print "hello, world. w=$w strlen(s)=".length($s)."\n";

File: lib/libqdev-bin-m68k/libqdev-1.2-bin-m68k/examples/mem_iloadseg/hello.c
/*
 * Hello World!
*/

#include <proto/exec.h>
#include <proto/dos.h>


 
LONG hello(void)
{
  struct DosLibrary *DOSBase;


  if ((DOSBase = (void *)OpenLibrary("dos.library", 36L)))
  {          
    FPuts(Output(), "Hello World!\n");

    CloseLibrary((void *)DOSBase);
  }

  return 0;
}
